docs/tokensdk.md

```diff
@@ -0,0 +1,156 @@
+* Allows developers to create `custom services` on top of the core API for specific application needs.
+
+The Fabric Token SDK stack consists of several layers:
+* Services: Pre-built functionalities like assembling transactions and selecting unspent tokens.
+* Token API: Provides a common abstraction for interacting with tokens across different backends.
+* Driver API: Translates generic token operations into backend-specific calls (e.g., Fabric).
+* Drivers: Define token representation, operations, and validation rules for specific implementations.
+
+The SDK leverages the Fabric Smart Client stack for complex workflows, secure storage, and event listening.
+
+The Fabric Token SDK has evolved beyond its initial focus on Hyperledger Fabric.
+It now empowers developers with the following capabilities across various platforms, including permissioned blockchains like Fabric:
+
+* **Tokenization Made Easy:** Create tokens representing any type of asset, be it physical or digital.
+* **Privacy by Design:** Select the appropriate privacy level for your specific use case, without modifying your application logic.
+* **Peer-to-Peer Transactions:** Orchestrate token transfers directly between users, streamlining the process.
+* **Atomic Swaps:** Facilitate secure exchanges of different tokens without relying on intermediaries.
+* **Transaction Auditing:** Review transactions before they are finalized, ensuring accuracy and compliance.
+* **Interoperability:** Connect with token systems on other blockchain networks, fostering broader ecosystems.
+* **Seamless Integration:** Add a token layer to existing applications, regardless of platform, with minimal effort.
+
+## Consumer Interaction Model
+
+The Token-SDK APIs are consumed by developers to perform various token-related operations.
+The developers must decide how to configure the Token-SDK to achieve the intended goals.
+Moreover, they are also responsible for defining the initial content of the datasource used by the Token-SDK.
+
+## Requirements and Use Cases
+
+Token-based applications that require:
+- Privacy;
+- Bankend agnostic;
+
+## Prerequisites, Dependencies, Dependents, and Incompatibilities
+
+The SDK leverages the following related projects:
+- [`Fabric Smart Client (FSC, for short)`](https://github.com/hyperledger-labs/fabric-smart-client): For complex workflows orchestration, secure storage, and event listening.
+- [`Idemix`](https://github.com/IBM/idemix): For the anonymous credentials.
+- [`Mathlib`](https://github.com/IBM/mathlib): For the elliptic curve math operations.
+
+The system administrator is responsible for preparing:
+
+- `Configuration`: FTS needs to be configured accordingly to the specific use-case.
+  FTS uses the FSC's `config service` to access its configuration.
+  An example of such a configuration can be found [`here`](./core-token.md).
+- `Data storage`: FTS requires an SQL data source to store relevant information for its functioning.
```

Comment:
How should the administrator provide access to the SQL source? Should this line include a reference to some relevant documentation?